packages/idos-sdk-js/src/lib/enclave-providers/iframe-enclave.ts

```diff
@@ -32,10 +32,11 @@ export class IframeEnclave implements EnclaveProvider {
       storage: { humanId, signerAddress, signerPublicKey },
     })) as StoredData;
 
-    if (encryptionPublicKey) return encryptionPublicKey;
+    while (!encryptionPublicKey) {
```

Comment:
This `while` triggered my wtf radar. Maybe it's worth our while to add a comment that this is meant to re-open the pop-up if the user closes it?

Come to think of it, how can the user back-off from this state now? (e.g., they decided they don't want to allow decryption anymore) I can only imagine they'll have to F5 the requesting page?

Reply:
Hard to tell! I feel a bit too uncertain, and I'd like @juliosantos to chime in on what made them want to put this here.